lib/elixir_jobs_web/templates/offer/_form.html.eex

```diff
@@ -30,7 +30,20 @@
 
     <fieldset>
       <div class="field-container">
+        <%= textarea(f, :summary,
+          rows: 3,
+          maxlength: 200,
+          id: "offer_summary",
+          class: class_with_error(f, :summary, "summary"),
+          placeholder: gettext("Offer summary (max 200 characters)")) %>
+        <%= error_tag(f, :summary) %>
+      </div>
+    </fieldset>
+
+    <fieldset>
+      <div class="field-container">
         <%= textarea(f, :description,
+          maxlength: 1000,
           id: "offer_description",
           class: class_with_error(f, :description, "description"),
           placeholder: gettext("Text should describe the offer itself. Remember, this text can potentially engage your candidates!")) %>
```

Comment:
And here something like `Text should describe the offer itself and engage your candidates (will be shown in offer details page)`